### DISCUSSION TITLE
Netbox 3.5.3 Importing Device Types with compilation error with thrown exception

### DISCUSSION THREAD

**Discussion by vncsinacio** [netbox-community/netbox#12927] [Help Wanted!]
<summary>
The Netbox 3.5.3 version has a compilation error (Python 3.10.6) for importing data, regardless of the file format (csv, yaml) for the platform. After the import, the error throws an exception as below:
</summary>
<p>
<p>
<p>

<b> <class 'TypeError'></b>
<b>unsupported operand type(s) for %: 'float' and 'decimal.Decimal'</b>

<p>
Therefore, even informing such data as defined for import, it presents a compilation problem and resulting in an error exception.

**Comment by vncsinacio** [comment]
The data reported for imports follows:
<br /> device_type,manufacturer,height,is_full_depth
  UPS APC RS 1500,APC,1,true

**Reply by gitAnhn** [reply]
Hi,

I'm having the same issue 
did you manage to solve this?

**Reply by kkthxbye-code** [reply]
There's no `height` field on the device_type model. There is a `u_height` though. He's also missing the `model` field, which I think he thinks is called `device_type`.

This works fine for example:

```
manufacturer,model,slug,u_height
APC,testmodel,testmodel,1
```

It's important to read what fields are supported, it's at the bottom of the screen. That being said, it's not supposed to throw an exception when the `u_height` field is missing, so feel free to create a bug report for that.

**Reply by gitAnhn** [reply]
thank you, friend
